spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -307,7 +308,71 @@ dataflow:> dataflow config server https://localhost:8443/
 
 By default, the REST endpoints (administration, management and health), as well
 as the Dashboard UI do not require authenticated access. However, authentication can
-be provided via http://oauth.net/2/[OAuth 2.0], thus allowing you to also integrate Spring Cloud
+be provided via:
+
+* http://oauth.net/2/[OAuth 2.0]
+* Basic Authentication
+
+[[getting-started-security-basic-authentication]]
+=== Basic Authentication
+
+https://en.wikipedia.org/wiki/Basic_access_authentication[Basic Authentication] can
+be enabled by adding the following to `application.yml` or via
+environment variables:
+
+[source,yaml]
+----
+security:
+  basic:
+    enabled: true                                                     # <1>
+    realm: Spring Cloud Data Flow                                     # <2>
+----
+
+In this use-case, the underlying Spring Boot will auto-create a user called _user_
+with an auto-generated password which will be printed out to the console upon startup.
+
+.Default Spring Boot user credentials
+image::images/dataflow-security-default-user.png[Default Spring Boot user credentials , scaledwidth="100%"]
+
+NOTE: Dedicated Basic Authentication support is available since Spring Cloud Data Flow 1.1.0. Also,
+please be aware of inherent issues of Basic Authentication and _logging out_, since the
+credentials are cached by the browser and simply browsing back to application pages will log you back in.
+
+If you need to define more than one file-based user account, please take a look at
+_File based authentication_.
+
+[[getting-started-security-file-based-authentication]]
+==== File based authentication
+
+By default Spring Boot allows you to only specify one single user. Spring Cloud
+Data Flow also supports the listing of more than one user in a configuration file,
+as described below. Each user must be assigned a password and a role:
+
+[source,yaml]
+----
+dataflow:
+  security:
+    authentication:
+      file:
+        enabled: true                                                 # <1>
+        users:                                                        # <2>
+          bob: bobspassword, ROLE_ADMIN                               # <3>
+          alice: alicepwd, ROLE_ADMIN
+----
+
+<1> Enables file based authentication
+<2> This is a yaml map of username to password
+<3> Each map `value` is made of a corresponding password and role
+
+NOTE: File based authentication support is available since Spring Cloud Data Flow 1.1.0.
```

Comment:
This snippet of doc will be added in 1.1.0+, so why bother include it?
To clarify: the pre-1.1.0 docs will not talk about this feature, so no need to clarify, is there?